readme.md

```diff
@@ -146,11 +220,11 @@ sections:
       - post
 ```
 
-And I have a document located at `content/posts/hello-world.md`, you'll find that document at `http://localhost:3000/posts/hello-world`.
+A document is located at `content/posts/hello-world.md`, lives at `http://localhost:3000/posts/hello-world`.
 
-> Notice that `hello-world` is considered the **relative** path here, we don't need to specify `content/posts/hello-world` because that's been configured in our `settings.yml`.
+Notice that `hello-world` is considered as the **relative** path, we don't need to specify `content/posts/hello-world` because it's already configured in our `settings.yml`.
```

Comment:
I think this section is still a bit confusing

Reply:
What's the point of specifying it's the relative path here?